sources/platform/integrations/ai/haystack.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Haystack integration
+sidebar_label: Haystack
+description: Learn how to integrate Apify with Haystack to work with web data in the Haystack ecosystem.
+sidebar_position: 1
+slug: /integrations/haystack
+---
+
+**Learn how to integrate Apify with Haystack to work with web data in the Haystack ecosystem.**
+
+---
+
+> For more information on Haystack, visit its [documentation](https://docs.haystack.deepset.ai/docs/intro).
```

Comment:
I would suggest removing the blockquote and moving the sentence introducing Haystack to the top.
This way we will: 

* introduce readers to Haystack itself
* present them with link where they can find more information on it (the docs link)
* and after all that we will move onto the example itself

It seems more logical this way

```suggestion
[Haystack](https://haystack.deepset.ai/) is an open source framework for building production-ready LLM applications, agents, advanced retrieval-augmented generative pipelines, and state-of-the-art search systems that work intelligently over large document collections. For more information on Haystack, visit its [documentation](https://docs.haystack.deepset.ai/docs/intro).
```